python/ray/tune/ray_trial_executor.py

```diff
@@ -436,11 +436,26 @@ def save(self, trial, storage=Checkpoint.DISK):
         """Saves the trial's state to a checkpoint."""
         trial._checkpoint.storage = storage
         trial._checkpoint.last_result = trial.last_result
+
+        subdir = ""
```

Comment:
this should be in save_to_disk as well (should not happen if storage == Checkpoint.MEMORY)